src/dev/bunHMRDevServer.ts

```
@@ -0,0 +1,794 @@
+            rebuildQueue: Array.from(this.rebuildQueue),
+            manifestKeys: Object.keys(this.manifest),
+            timestamp: Date.now()
+          }), {
+            headers: { 'Content-Type': 'application/json' }
+          });
+          
+        default:
+          // Try to serve static files - CSS, JS, images, etc.
+          const filePath = `./example/build${pathname}`;
+          try {
+            const file = Bun.file(filePath);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)